gnoland/website/main.go

```diff
@@ -41,14 +42,15 @@ func init() {
 	flag.StringVar(&flags.bindAddr, "bind", "127.0.0.1:8888", "server listening address")
 	flag.StringVar(&flags.captchaSite, "captcha-site", "", "recaptcha site key (if empty, captcha are disabled")
 	flag.StringVar(&flags.faucetURL, "faucet-url", "http://localhost:5050", "faucet server URL")
+	flag.StringVar(&flags.homeDir, "home", "./gnoland/website/views", "home index file directory")
```

Comment:
can you rename it to something else than "home", maybe like this?

```suggestion
	flag.StringVar(&flags.viewsDir, "views", "./gnoland/website/views", "views directory location")
```

Reply:
Sure,  I will close the one and make a new one. There are other changes in main.go